Date/TimeZone.php

```diff
@@ -495,7 +494,7 @@ function setDefault($id)
      * @access   public
      * @see      Date::setTZByID(), Date_TimeZone::Date_TimeZone()
      */
-    function isValidID($ps_id)
+    static function isValidID($ps_id)
```

Comment:
It's the same problem and the solution I mention in the previous comment thread.